gelfclient/client.py

```diff
@@ -44,6 +50,9 @@ def log(self, _fields_dict = {}, **fields_named):
                 message['host'] = message['source']
             else:
                 message['host'] = self.source
+        if 'level' not in message:
+            message['level'] = LogLevel.alert
```

Comment:
I just installed a Graylog server to test out the log level. Even though their GELF spec says that 1 (ALERT) is the default, it is not set by the server if not provided by the client. So I think keep it here but change to ```message['level'] = 1 # alert```